src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,96 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import DetailFeed from './components/DetailFeed';
+import DetailProfile from './components/DetailProfile';
+import DetailComment from './components/DetailComment';
 
 const Detail = () => {
-  return <div>Detail</div>;
+  const [feedData, setFeedData] = useState([]);
+
+  useEffect(() => {
+    fetch('data/detailfeed.json')
+      .then(response => response.json())
+      .then(result => setFeedData(result));
+  });
+
+  return (
+    <>
+      <DetailNavBox>
+        <DetailNav>
+          <DetailNavBtn>10평대</DetailNavBtn>
+          <DetailNavBtn>빌라 & 연립</DetailNavBtn>
+        </DetailNav>
+      </DetailNavBox>
+      <Container>
+        <FeedBox>
+          {feedData.map((feeds, idx) => {
+            return <DetailFeed key={idx} feeds={feeds} />;
+          })}
```

Comment:
feedData 배열 내부의 각각 요소들이 feeds가 아닌 feed여야 할 것 같습니다!
그리고 key prop에 index 사용은 정말 어쩔 수 없는 경우가 아니면 지양해주세요!
```suggestion
          {feeds.map((feed, idx) => {
            return <DetailFeed key={feed.id} feed={feed} />;
          })}
```